src/lib/story-intents.ts

```diff
@@ -10,7 +10,7 @@ import { Story } from '../@types/storybook-emotion-10-fixes';
  * Make sure all props are configurable in storybook's controls table.
  *
  */
-export const asPlayground = (story: Story) => {
+export const asPlayground = (story: Story<any>) => {
```

Comment:
Out of interest, what does having the `<any>` and the corresponding `Story<TilesProps>` do? Is it so that typescript knows what args to expect? Should we update the recommendations to match this? And is it worth adding an eslint ignore so that we don't get this warning?

Reply:
TS was really not happy with:

```typescript
const Template: Story = (args: TilesProps) => ( ... )
```
saying that `Story<Args>` did not contain `columns` that `TilesProps` requires (which is true). 

`const Template: Story<TilesProps>` resolve that, but then `asChromaticStory`/`asPlayground` complained that the `Story<Args>` they expected were not compatible with `Story<TilesProps>` (again true).

so now they just take a `Story` with any old thing passed into the main function.

this might not be the right approach...

Reply:
We could make `asChromaticStory` / `asPlayground` accept a generic type that gets passed to the `Story` type in their signature

e.g.

```ts
// story-intents.ts

export const asPlayground = <T>(story: Story<T>) => {
```

Reply:
```suggestion
export const asPlayground = <Args>(story: Story<Args>) => {
```